### DISCUSSION TITLE
How to write to a log file, or to get SWS to write its logs to /var/log/syslog?

### DISCUSSION THREAD

**Discussion by ramsamba** [static-web-server/static-web-server#204] [Q&A]
Great job on SWS. Thank you

I do not find any option to write the logs that appear on screen with the --log-level info option.
Is there any way of writing SWS's output to a log file in /var/log/, or to /var/log/syslog?

**Comment by joseluisq** [comment]
SWS does not support file logging. However, in Linux/BSDs you could achieve that by redirecting SWS stdout output to a file.

```bash
touch /var/log/sws.log
chmod 640 /var/log/sws.log
chown my-user:my-group /var/log/sws.log

static-web-server --port 8787 --root ./my-public-dir  --log-level "info" >> /var/log/sws.log 2>&1
```

**Comment by hanscees** [comment]
if it runs in a docker container 

docker logs -f sws-web-1 &> /var/log/swsweb.txt &
